dashboard/views.py

```diff
@@ -66,9 +66,63 @@ def get(self, request):
 
 class WarningsInstance_View(View):
 
-    def get(self, request):
+    def get(self, request, **kwargs):
+
+        cap_id = kwargs['cap_id']
+        cap_instance = CAPInstance.objects.get(cap_id=cap_id)
+
+        # Grab Masterlist Atrributes
+        brokenlist = self.get_broken_urls(cap_instance)
+        retaglist = self.get_retag(cap_instance)
+        nimlist = self.get_nim(cap_instance)
+
+        context = {'date': cap_instance.date, 'brokenlist':brokenlist[:5], 'retaglist': retaglist[:5], 'nimlist': nimlist[:5]}
```

Comment:
Exactly - I can do this!

Reply:
Oh actually in this instance - it is only for demo purposes - this will be changed to include the whole list however in the homepage view class this needs to be implemented